src/cli/serve/serve.js

```diff
@@ -283,18 +280,11 @@ export default function (program) {
       .option(
         '--run-examples',
         'Adds plugin paths for all the Kibana example plugins and runs with no base path'
-      );
-  }
-
-  if (isServerlessCapableDistribution()) {
-    command
-      .option(
-        '--serverless',
-        'Start Kibana in the most recent serverless project mode, (default is es)'
       )
       .option(
-        '--serverless <oblt|security|es>',
-        'Start Kibana in a specific serverless project mode'
+        '--serverless [oblt|security|es]',
```

Comment:
the plain `--serverless` option was handling all commands, and we were never making it to the values here.

Looking at `commander`'s documentation, using `[value]` allows us to specify that they can be optional.